src/infrastructure/figma/figma-service.ts

```diff
@@ -153,7 +159,10 @@ export class FigmaService {
 
 		if (response.errors?.length > 0) {
 			const errorMessage = response.errors.map((err) => err.error).join('|');
-			getLogger().error(errorMessage, 'Created dev resources with errors');
+			getLogger().error(
```

Comment:
**question:** Do we need to throw an error after logging it? 

If no, consider logging just a message without an error creation:

```ts
getLogger().error('Created dev resources with errors', response.errors);
```

Reply:
Agree that we don't need to create an error, but I think pino's logging API requires the object first unlike console.log ([docs](https://github.com/pinojs/pino/blob/master/docs/api.md#loggererrormergingobject-message-interpolationvalues)). You might also need to wrap the errors in an object so they get merged into the JSON log structure correctly

```typescript
getLogger().error({errors: response.errors}, 'Created dev resources with errors');
```

Reply:
I believe it needs to be inside an `err` property:
https://getpino.io/#/docs/api?id=mergingobject-object

so `{err: response.errors}`

Reply:
I think those docs are just saying if you pass an `Error` as the first argument it'll get merged into the JSON log object under the `err` key. The first argument can be any object with arbitrary keys, and those will get merged into the final log object.